modules/EvseManager/IECStateMachine.cpp

```diff
@@ -505,7 +505,8 @@ void IECStateMachine::connector_force_unlock() {
         cp = cp_state;
     }
 
-    if (cp == RawCPState::B or cp == RawCPState::C) {
+    // Unlocking in state C violates regulations and risks the user.
```

Comment:
I can only speculate but the reasoning why its only force unlocked in B and C is because we already unlock (without the force) when we enter state A (see same file)

Reply:
I would think that in general force unlock should always unlock but before doing so open the contactors (and wait for state A or B)